CHANGELOG.md

```diff
@@ -7,8 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0] - 07-01-21
+### Changed
+- Blinding factor provided to create obfuscated notes
+- Transparent note with constant blinding factor
```

Comment:
This is not necessary to add in the changelog, since as discussed we don't expose it in any way, and does not affect the API or the internal behavior, so it shouldn't mentioned in the changelog. It's just an implementation detail.

Reply:
Fixed